lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -0,0 +1,12 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi < DelegateClass(ActiveModelSerializers::Adapter::JsonApi)
+        def initialize(serializer, options = {})
```

Comment:
Since I need to pass the non-deprecated adapter instance to `super` for delegation to work, I'm not sure I can do that (or I don't know how to do it).